media-plugins/x42-plugins/x42-plugins-20200714.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Collection of LV2 plugins"
+HOMEPAGE="https://github.com/x42/x42-plugins"
+SRC_URI="http://gareus.org/misc/x42-plugins/${P}.tar.xz"
+KEYWORDS="~amd64"
+LICENSE="GPL-2"
+SLOT="0"
+RESTRICT="mirror"
+
+RDEPEND="dev-libs/glib
```

Comment:
I assume there are minimum version requirements for a lot of these deps... if upstream documents any of them they should be added here, if they don't I guess we don't have to care for now.

Reply:
I'll do a check if I can find some specific versions.

Reply:
Checked this, there are only minimal versions defined for LV2, not for anything else. For LV2 it's version 1.8.1 which is long out of the gentoo tree, so there are effectively no version requirements.